### PR TITLE
 changes about page and contact page

### PR DESCRIPTION
**Pull request by Kendiya242**

### CODE REVIEW

**Review by AndalatuGH — APPROVED** [review]
looks good

File: views/pages/contact-page.ejs
Comment (dinaolmelak) [inline views/pages/contact-page.ejs:1]: Well Done! 👍

File: views/pages/index.ejs
Comment (dinaolmelak) [inline views/pages/index.ejs:1]: Great Job!, Thank you

**Review by dinaolmelak — APPROVED** [review]
LGTM

**Pull request merged**